Backend/src/main/java/ktsnvt/tim1/services/UserService.java

```diff
@@ -21,21 +21,24 @@
     @Autowired
     private UserMapper userMapper;
 
-    public UserDTO editUser(UserDTO user) throws EntityNotValidException, EntityNotFoundException{
+    public UserDTO editUser(UserDTO user) throws EntityNotValidException {
+        User registeredUser = (User) SecurityContextHolder.getContext().getAuthentication().getPrincipal();
```

Comment:
Name "user" would be more suited for this variable (currently named "registeredUser").